src/components/Footer/footer.scss

```
@@ -0,0 +1,49 @@
+@import "../../assets/scss/variables";
+
+.footer {
+    background-color: $dark-grey;
+    font-weight: 250;
+    color: $grey;
+    padding: 1.5rem;
+    box-shadow: 0px -20px 24px 11px rgb(69 75 105 / 26%);
```

O footer ficou muito bom, mas acho que talvez esta sombra pode atrapalhar o restante do conteúdo. Poderíamos removê-lo:
```suggestion
```
ou adicioar uma margem superior:
```suggestion
    margin-top: 3rem; /* testar */
    box-shadow: 0px -20px 24px 11px rgb(69 75 105 / 26%);
```